shell/platform/fuchsia/flutter/flatland_connection.h

```diff
@@ -89,10 +93,13 @@ class FlatlandConnection final {
   // You should always lock mutex_ before touching anything in this struct
   struct {
     std::mutex mutex_;
-    FireCallbackCallback fire_callback_;
-    bool first_present_called_ = false;
-    bool on_next_frame_pending_ = false;
-    fml::TimePoint next_presentation_time_;
+    bool first_feedback_received_ = false;
+    FireCallbackCallback pending_fire_callback_;
+    uint32_t present_credits_ = 1;
+    std::queue<fml::TimePoint> next_presentation_times_;
```

Comment:
Nit: queue first, then other fields, then boolean last (more readable, and fields pack into structure better)

Reply:
Done.